src/common/src/array/map_array.rs

```diff
@@ -431,7 +431,7 @@ impl ToText for MapRef<'_> {
                 let key = key.to_text();
                 let value = value.to_text();
                 // TODO: consider quote like list and struct
-                f(&format_args!("\"{}\":{}", key, value))
+                f(&format_args!("{}:{}", key, value))
```

Comment:
Changed this because I feel quoting integer key doesn't look good. But not sure whether to quote string keys. (array's text repr doesn't quote string)